xslt/Channels.xsl

```diff
@@ -1,18 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<xsl:stylesheet version="3.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:stylesheet version="1.1" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:ome="http://www.openmicroscopy.org/Schemas/OME/2016-06">
 
     <!-- /Metadata/DisplaySetting/Channels/Channel/DyeName => /OME/Image/Channel@Fluorophore -->
-    <xsl:template match="DyeName">
-       <xsl:attribute name="Fluorophore">
-           <xsl:value-of select="."/>
-       </xsl:attribute>
-    </xsl:template>
+<!--    <xsl:template match="DyeName">-->
+<!--       <xsl:attribute name="Fluorophore">-->
+<!--           <xsl:value-of select="."/>-->
+<!--       </xsl:attribute>-->
+<!--    </xsl:template>-->
```

Comment:
This one appears to be valid so I'd add it back in and see how it goes. My cheat is to look for the tag in "resources/example-ome.xml" which was generated from the OME schema (if memory serves)

Reply:
Resolved in commit 9ee7971c2dea4c4b64ca97158befe3dcf84fb243.